bigtable/tests/system.py

```diff
@@ -89,12 +89,12 @@ def setUpModule():
     if not Config.IN_EMULATOR:
         retry = RetryErrors(GrpcRendezvous,
                             error_predicate=_retry_on_unavailable)
-        instances, failed_locations = retry(Config.CLIENT.list_instances)()
+        instances_response = retry(Config.CLIENT.list_instances)()
```

Comment:
Please revert: this patch is out-of-sync with `master`.  #5420 restored returning actual `Instance` objects, rather than protos.

Reply:
@tseaver This is not working with GAPIC api. GAPIC api response the ListInstancesResponse instance. It gives an error ListInstancesResponse is not iterable.

Reply:
@aneepct You need to sync with master.  `Client.list_instances()` returns a tuple (list of `Instance`, list of failed locations).